pkg/apis/ignite/types.go

```diff
@@ -183,12 +172,12 @@ type VMSpec struct {
 }
 
 type VMImageSpec struct {
-	OCIClaim OCIImageClaim `json:"ociClaim"`
+	OCIRef meta.OCIImageRef `json:"oci"`
 }
 
 type VMKernelSpec struct {
-	OCIClaim OCIImageClaim `json:"ociClaim"`
-	CmdLine  string        `json:"cmdLine,omitempty"`
+	OCIRef  meta.OCIImageRef `json:"oci"`
```

Comment:
Generally, you always want to match the field name with the JSON tag to avoid so-called "OpenAPI violations".
Let's change this name to avoid those?

Reply:
Changed in 9d6ab9e.